docs(web-modeler): document file download and export options (#3309)
camunda/camunda-docs: docs/components/modeler/web-modeler/file-download.md

---
id: file-download
title: File download
description: Learn how to download files from Web Modeler and export diagrams in different image formats.
---

import SingleFileDownloadImg from './img/download/single-file-download.png'
import MultiFileDownloadImg from './img/download/multi-file-download.png'
import DownloadBpmnImg from './img/download/download-bpmn.png'
import DownloadDmnImg from './img/download/download-dmn.png'
import DownloadFormImg from './img/download/download-form.png'
import DownloadConnectorTemplateImg from './img/download/download-connector-template.png'

<span class="badge badge--cloud">Camunda 8 only</span>

> You can download all the files stored in Web Modeler to your local computer.

Reviewer: ```suggestion
You can download all the files stored in Web Modeler to your local computer for offline editing or integration into version control.
```
introduces the main use cases for download

> For BPMN diagrams, you also have the option to export them in different image formats.

Reviewer: ```suggestion
You may also export BPMN diagrams in different image formats.
```
suggestion for brevity
